test_files/boundaries.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Would be good to copy the pattern for other types where we check fetching a single object and also multiple objects. Separate test file for each.